src/plugins/intel_cpu/src/transformations/snippets/x64/pass/brgemm_to_brgemm_cpu.cpp

```diff
@@ -106,7 +107,7 @@ pass::BrgemmToBrgemmCPU::BrgemmToBrgemmCPU() {
                                                          brgemm_in0_desc->get_layout(), std::vector<size_t>{}, brgemm_out_desc->get_layout());
                 set_full_port_desc(brgemm_repacking->output(1));
                 set_full_port_desc(brgemm_cpu->input(2));
-            } else if (one_of(element_type_a, ov::element::u8, ov::element::bf16)) {
+            } else if (element_type_a == ov::element::u8 || (element_type_a == ov::element::bf16 && isBF16Supported)) {
```

Comment:
Could you please clarify why we need this change now?
If we check for `amx` and `bf16` instruction support here, should we also check for `vnni` instruction support for quantized case?

Reply:
I've found interesting [something](https://github.com/openvinotoolkit/oneDNN/blob/8aec0b5b29d61bd27e561bf1c8f300e0d56d1ac1/src/cpu/x64/brgemm/brgemm_utils.cpp#L844). Should we update condition for `with_comp` to `element_type_a == ov::element::i8 && !with_amx && !avx2_vnni_2`?

Probably, we also should update it in emitters